internal/source/vmware/vmware_sync.go

```diff
@@ -21,11 +21,17 @@ func (vc *VmwareSource) syncTags(nbi *inventory.NetboxInventory) error {
 	objectNames2NBTags := make(map[string][]*objects.Tag)
 	for objectName, tags := range vc.Object2Tags {
 		for _, tag := range tags {
+			var description string
+			if tag.Description != "" {
+				description = fmt.Printf("Tag synced from vmware:%s", tag.Description)
+			} else {
+				description = fmt.Printf("Tag synced from vmware")
```

Comment:
Sorry my bad, this should be `description = "Tag synced from vmware"`

Reply:
done! 👍